p2p_wallet/AppDelegate.swift

```diff
@@ -120,14 +120,19 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 
     func application(
-        _: UIApplication,
+        _ application: UIApplication,
         open url: URL,
         options: [UIApplication.OpenURLOptionsKey: Any] = [:]
     ) -> Bool {
-        var result = false
-        Broadcaster.notify(AppUrlHandler.self) { result = result || $0.handle(url: url, options: options) }
+        var isGoogleServiceUrlHanded = false
+        Broadcaster.notify(AppUrlHandler.self) { isGoogleServiceUrlHanded = isGoogleServiceUrlHanded || $0.handle(url: url, options: options) }
+        if isGoogleServiceUrlHanded {
```

Comment:
is it ok that methods stops here with `true` and never reach to 133 and 135 lines?

Reply:
It's ok, because if the url was handed by som provider, the next provider doesn't need to handle it. Google Service only handles links that related to themself.

Reply:
But it has `return proxyAppDelegate.application(application, open: url, options: options)` on 135
proxyAppDelegate may contain important logic as well